packages/plugins/test/mocha/foundry/utils.spec.ts

```diff
@@ -178,7 +178,7 @@ describe('Utils', async () => {
       // Fifth contract:
       expect(
         existsSync(
-          `${artifactFolder}/test/deeper/DuplicateContractName.sol/DuplicateContractName.json`
+          `${artifactFolder}/deep/deeper/DuplicateContractName.sol/DuplicateContractName.json`
```

Comment:
This fixes a separate issue where one of our tests was failing because an update in foundry caused the artifact to be output in a different location. It seems like this is just them fixing a bug, and updating this value caused the entire test to pass, so I don't think this change has any other effects, but let me know if you disagree.